docs/tutorials/release-using-channels.md

```diff
@@ -0,0 +1,63 @@
+> !!! This documentation is in "beta" and needed to be tested !!!
+
+# Release using channels / Auto-updates with channels
+
+## Description
+Channels are useful to distribute "beta" or "alpha" releases of your application to a choosen set of users. This allow to test an application before release it as "stable".
+
+Users which receive "beta" version will get "stable" versions too. Otherwise users who doesn't wanna "beta" will only get "stable" releases.
+
+They are tree channels level ordered by stability :
+1. "stable", your application is stable and this is the default one (example: 1.3.2)
+2. "beta" which mean your application works, but should have some bugs (example: 1.3.2-beta)
+3. "alpha" which mean your application is not stable and in active development (example: 1.3.2-alpha)
+
+
+## Configuration
+To release using channels, you should config electron-builder and define the channels to use in client side.
+
+### Electron-Builder
+By default (without using channels), all application releases use the "stable" channel.
+
+If you want to use channels, you should add this to your package.json:
+
+```
+"version": "x.x.x-beta",
+...
+"build": {
+  "generateUpdatesFilesForAllChannels": true,
+  ...
+}
+```
+
+> Note: `allowDowngrade` is automatically set to `true` when `generateUpdatesFilesForAllChannels = true`, so you doesn't need to set it.
+
+All you have to do to release using channels is to define the channel in the version tag of the `package.json`. Add "-beta" or "-alpha" (nothing for stable) to automatically build for the related channel.
+
+
+### Your application
+All you need to do here is to define which channel the user will receive.
+
+`autoUpdater.setFeedURL({url: 'https://www.yoursite.com', channel: 'beta'});`
```

Comment:
Now user can set `channel`, `setFeedURL` not required, right?

Reply:
I'm confused. Where do we set `channel` ?

Reply:
Oh this: https://github.com/electron-userland/electron-builder/blob/ba4809aeb3bd3b27f66ee684c3e6e8ab0cf55186/packages/electron-updater/src/AppUpdater.ts#L73. Needs to be documented

Reply:
@kwent and @develar Thanks for feedback. I've remove the "old" `setFeedURL` and replace it with `autoUpdater.channel()`. I will add the doc for `channel()` too.